packages/create-snap/package.json

```diff
@@ -49,15 +49,15 @@
     "lint:dependencies": "depcheck",
     "lint:eslint": "eslint . --cache --ext js,ts,jsx,tsx",
     "lint:fix": "yarn lint:eslint --fix && yarn lint:misc --write",
-    "lint:misc": "prettier --no-error-on-unmatched-pattern --loglevel warn \"**/*.json\" \"**/*.md\" \"**/*.html\" \"!CHANGELOG.md\" --ignore-path ../../.gitignore",
+    "lint:misc": "prettier --no-error-on-unmatched-pattern --log-level warn \"**/*.json\" \"**/*.md\" \"**/*.html\" \"!CHANGELOG.md\" --ignore-path ../../.gitignore",
     "publish:package": "../../scripts/publish-package.sh",
     "publish:preview": "yarn npm publish --tag preview",
     "since-latest-release": "../../scripts/since-latest-release.sh",
-    "test": "jest --reporters=jest-silent-reporter",
-    "test:clean": "jest --clearCache",
+    "test": "node --experimental-vm-modules $(yarn bin jest) --reporters=jest-silent-reporter",
```

Comment:
Prettier v3's main entry point (which we use for `resolveConfig`) uses dynamic imports, so we need to either:

- Run Jest with `--experimental-vm-modules`, or
- Reimplement the config resolution logic ourselves.